### DISCUSSION TITLE
Leech stucked after full download completion

### DISCUSSION THREAD

**Discussion by JOHN748** [anasty17/mirror-leech-telegram-bot#1261] [Q&A]
Am getting stucked so many times at leeching after the file is downloaded and get stucked its not uploading. If I restart and use it'll working but after some time again tgis problem occurs. This is the bot log.

![image](https://user-images.githubusercontent.com/48284893/205269988-ddf65439-b9a6-4ab0-b653-8ab27a91c0ee.png)

**Comment by anasty17** [comment]
Send full logs = log.txt

**Reply by JOHN748** [reply]
Here is the full log : [log.txt](https://dl15.debrid.link/dl/8srld7b1217f8cf14503aae53ff7544b35845ceedd6a/log.txt)

**Reply by anasty17** [reply]
Read 2nd line (warning)
